lib/core/utilities.js

```diff
@@ -83,3 +83,9 @@ export function decodeValue (val) {
   // Return as is
   return val
 }
+
+export class ExpiredSessionError extends Error {
```

Comment:
I would suggest naming it to something like `ExpiredAuthSessionError` or `AuthExpiredSessionError`

Reply:
Good idea! That's more explicit